src/tribler-gui/tribler_gui/tests/test_gui.py

```diff
@@ -25,8 +20,9 @@
 from tribler_gui.tribler_window import TriblerWindow
 from tribler_gui.widgets.loading_list_item import LoadingListItem
 
-if sys.platform.startswith('win'):
-    asyncio.set_event_loop(asyncio.SelectorEventLoop())
+RUN_TRIBLER_PY = Path(tribler_gui.__file__).parent.parent.parent / "run_tribler.py"
+
+SAMPLE_STATEDIR = RUN_TRIBLER_PY.parent / "tribler-core/tribler_core/tests/tools/data/state_dir_guitests"
```

Comment:
Is this `SAMPLE_STATEDIR` actually used?

Reply:
Oops, thanks for catching it. Forgot from the previous iteration